components/config/opts.rs

```diff
@@ -791,6 +784,10 @@ pub fn from_cmdline_args(args: &[String]) -> ArgumentParsingResult {
         }
     };
 
+    let max_session_history = opt_match.opt_str("max-session-history").map(|max| {
+        max.parse().unwrap_or_else(|err| args_fail(&format!("Error parsing option: --max-session-history ({})", err)))
+    }).unwrap_or(256);
```

Comment:
Do we have a way to set a numeric preference from the command line? AFAICT we can only set strings and booleans. We use the command line for running the wpt suite with `--binary-arg=--max-session-history=N`.